### ISSUE TITLE
Deprecation in collections call

### ISSUE THREAD

**Issue by Cajuteq** [ovh/python-ovh#96]
Hi, using python3.9 with poetry, I found out running pytest that the collections call used in ovh are deprecated and will stop working with 3.10

```
[...] lib/python3.9/site-packages/ovh/vendor/requests/packages/urllib3/_collections.py:2: DeprecationWarning: 
Using or importing the ABCs from 'collections' instead of from 'collections.abc' is deprecated since Python 3.3, 
and in 3.10 it will stop working
    from collections import Mapping, MutableMapping

-- Docs: https://docs.pytest.org/en/stable/warnings.html
```

PyPI currently says ovh package is compatible with at most python3.6 which according to https://www.python.org/dev/peps/pep-0494/ will stop being updated on 2021-12.

Is there any work on making this package compatible with newer python version. It would be good to be able to continue using it under Python 3.10 !

**Comment by rbeuque74** [comment]
Hello,
Sorry for the delay,
This issue will be fixed in `v1.0.0`.

**Comment by rbeuque74** [comment]
Fixed with #108.
Released in https://github.com/ovh/python-ovh/releases/tag/v1.0.0
